pygsti/tools/optools.py

```diff
@@ -2495,204 +2494,136 @@ def compute_best_case_gauge_transform(gate_mx, target_gate_mx, return_all=False)
-
-                    #Add real eigenvalues and vectors
-                    std_evals.extend([ev] * dim)
-                    std_evecs.extend([U[:, i] for i in info['indices']])
-
-                else:  # complex eigenvalue case - should have conjugate pair info
-                    #Ensure blocks for conjugate-pairs of eigenvalues follow one after another and
-                    # corresponding eigenvectors (e.g. the first of each block) are conjugate pairs
-                    # (this is already done in the eigenspace construction)
-                    assert(len(info['conj_pair_indices']) == dim)
-                    std_evals.extend([ev] * dim)
-                    std_evals.extend([_np.conjugate(ev)] * dim)
-                    std_evecs.extend([U[:, i] for i in info['indices']])
-                    std_evecs.extend([U[:, i] for i in info['conj_pair_indices']])
-
-            return _np.array(std_evals), _np.array(std_evecs).T
-
-        #Create "gate_tilde" which has the eigenvectors of gate_mx around the matched eigenvalues of target_gate_mx
-        # Doing this essentially decouples the problem of eigenvalue matching from the rest of the task -
-        # after gate_tilde is created, it and target_gate_mx have exactly the *same* eigenvalues.
-        evals_tgt, Utgt = _np.linalg.eig(target_gate_mx)
-        evals_gate, Uop = _np.linalg.eig(gate_mx)
-        pairs = _mt.minweight_match_realmxeigs(evals_gate, evals_tgt)
-        replace_evals = _np.array([evals_tgt[j] for _, j in pairs])
-        gate_tilde = _np.dot(Uop, _np.dot(_np.diag(replace_evals), _np.linalg.inv(Uop)))
-
-        #Create "standard diagonalizations" of gate_tilde and target_gate_mx, which give
-        # sort the eigenvalues and ensure eigenvectors occur in *corresponding* conjugate pairs
-        # (e.g. even when evals +1j and -1j have multiplicity 4, the first 4-D eigenspace, the
-        evals_tgt, Utgt = standard_diag(target_gate_mx)
-        evals_tilde, Uop = standard_diag(gate_tilde)
-        assert(_np.allclose(evals_tgt, evals_tilde))
-
-        #Update Utgt so that Utgt * inv_Uop is close to the identity
-        kite = _mt.compute_kite(evals_tgt)  # evals are grouped by standard_diag, so this works
-        D_prior_to_proj = _np.dot(_np.linalg.inv(Utgt), Uop)
-        #print("D prior to projection to ",kite," kite:"); _mt.print_mx(D_prior_to_proj)
-        D = _mt.project_onto_kite(D_prior_to_proj, kite)
-        start = 0
-        for i, k in enumerate(kite):
-            slc = slice(start, start + k)
-            dstart = start + k
-            for kk in kite[i + 1:]:
-                if k == kk and _np.isclose(evals_tgt[start], evals_tgt[dstart].conj()):  # conjugate block!
-                    dslc = slice(dstart, dstart + kk)
-                    # enforce block conjugacy needed to retain Uproj conjugacy structure
-                    D[dslc, dslc] = D[slc, slc].conj()
+    def _get_eigenspace_pairs(mx, tol=1e-6):
```

Comment:
All changes past this point are from simplifying an ``if True: ...`` block to remove the unreachable code.